src/Core/Framework/App/Lifecycle/Persister/ScriptPersister.php

```diff
@@ -105,10 +105,11 @@ public function refresh(): void
         $criteria->setTitle('app-scripts::refresh');
         $criteria->addFilter(new EqualsFilter('active', true));
 
-        $apps = $this->appRepository->search($criteria, Context::createDefaultContext())->getEntities();
+        /** @var array<string> $apps */
```

Comment:
I think the hard coded type can be removed, as it is already present in the return type of the [`getIds`](https://github.com/shopware/shopware/blob/trunk/src/Core/Framework/DataAbstractionLayer/Search/IdSearchResult.php#L58) method.
```suggestion
```

Reply:
The return type of getIds() is not `array<string>` but `list<string>|list<array<string, string>>`
This would lead to phpstan issues. Take a look at the other id-fetches in this or other shopware files